archivers/upx-devel/Portfile

```diff
@@ -0,0 +1,45 @@
+# $Id$
+
+PortSystem       1.0
+PortGroup        github 1.0
+
+github.setup     upx upx 5c0340f
+
+name             upx-devel
+version          20161019
+categories       archivers
+maintainers      openmaintainer
+license          GPL-2+
+
+description      compress or expand executable files
+long_description UPX is a free, portable, extendable, high-performance \
+                 executable packer for several executable formats
+homepage         https://upx.github.io/
+platforms        darwin
+checksums        ${distname}${extract.suffix} \
+                 rmd160  205df5485c93e452c591402cc579baad785d8ffd \
+                 sha256 28c859898e61025f65f4985162de2a1fdaa221fd46453e7b2ea4facd7a67060e 
+
+use_configure    no
+
+depends_lib      port:ucl port:zlib
+
+fetch.type          git
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init --recursive"
+}
+
+post-configure {
+    # Ensure that the correct compiler is used
```

Comment:
This comment seems misleading; you're ensuring the right flags are used, but not the right compiler.